matcher.rb

```diff
@@ -62,7 +62,12 @@ def match!
         if available_athletes.length > 0
           match_values = available_athletes.map{ |a| selected_athlete.matchup(a) }.compact.sort_by{ |x| x[:heuristic] }
           best_match = match_values.last
-          matches << [selected_athlete, best_match[:athlete]]
+          
+            max_heuristic = best_match[:heuristic] if best_match[:heuristic] > max_heuristic
+          
+          confidence_percentage = (best_match[:heuristic] / max_heuristic) * 100
```

Comment:
This could probably be better expressed if the max heuristic was just the max possible value attainable here: https://github.com/Formless/youth-challenge-matching/blob/4779444e3e66bc4604fed3bf0e505fce3859effb/matcher.rb#L125-L153